modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/css/_balloontoolbar.scss

```diff
@@ -0,0 +1,53 @@
+.lfr-balloon-editor {
+	&.cke_balloon.cke_balloontoolbar {
+		background: white;
+		border-color: #e7e7ed;
+		box-shadow: 0 8px 16px rgba(0, 0, 0, 0.2);
+		.cke_balloon_triangle_outer.cke_balloon_triangle_bottom {
```

Comment:
Minor SF: there should be a blank row between sibling classes and between styles and first class.